seqr/management/commands/lift_project_to_hg38_tests.py

```diff
@@ -0,0 +1,235 @@
+#-*- coding: utf-8 -*-
+import mock
+import __builtin__
+from copy import deepcopy
+from django.core.management.base import CommandError
+from seqr.models import Family
+from pyliftover.liftover import LiftOver
+from seqr.views.utils.test_utils import VARIANTS, SINGLE_VARIANT
+
+from django.core.management import call_command
+from django.test import TestCase
+
+PROJECT_NAME = u'1kg project n\u00e5me with uni\u00e7\u00f8de'
+PROJECT_GUID = 'R0001_1kg'
+ELASTICSEARCH_INDEX = 'test_index'
+INDEX_METADATA = {
+    "gencodeVersion": "25",
+    "hail_version": "0.2.24",
+    "genomeVersion": "38",
+    "sampleType": "WES",
+    "sourceFilePath": "test_index_alias_1_path.vcf.gz",
+}
+SAMPLE_IDS = ["NA19679", "NA19675_1", "NA19678", "HG00731", "HG00732", "HG00733"]
+
+liftover_to_38 = LiftOver('hg19', 'hg38')
+
+LIFT_MAP = {
+    21003343353L: [('chr21', 3343400)],
+    1248367227L: [('chr1', 248203925)],
+    1001562437L: [('chr1',   1627057)],
+    1001560662L: [('chr1',  46394160)],
+}
+
+
+def mock_convert_coordinate(chrom, pos):
+    pos = int(chrom.replace('chr', ''))*int(1e9) + pos
+    if pos in LIFT_MAP.keys():
+        return(LIFT_MAP[pos])
```

Comment:
This works for the given test cases, but you want this function to always return something, since in general in python its bad to have a function that in some cases has a return value and in some cases doesn't. You could just do `return LIFT_MAP.get(pos)`

Reply:
It's fair. I assumed it returns None when the key doesn't exist. I could be wrong. Updated.

Reply:
Yeah, in python if you reach the end of a function without a specific return it just doesn't return anything, and you usually get a runtime error